api/controllers/user.go

```diff
@@ -0,0 +1,38 @@
+package controllers
+
+import (
+  "net/http"
+  "strings"
+  "github.com/gin-gonic/gin"
+  "github.com/charmbracelet/log"
+
+  "github.com/Salz-Team/salz/api/db"
+)
+
+func GetUser(c *gin.Context, dbHandler db.ApiDBHandler, authHandler db.AuthDBHandler) {
```

Comment:
This made me think of whether we want to allow users to see the bot results of other users. @Lugarun thoughts?

Reply:
@Lugarun mr product manager plz advise

I think in halite it was possible to download anybody's matches, but i may be remembering wrong

Reply:
Sorry for the late reply, was busy consulting my decks to see which kpi's driver customer engagement.

I think users should be able to see the bot results of other users, it was really nice to see the decisions the best bots were making in matches in halite. This also gives more transparency on why #1 is #1.

Reply:
`/users/me` will be kept private for the person's own data (maybe we return more private info?)

I think we can expose `/users/` and `/users/{userId}` as public endpoints though.